src/App.tsx

```diff
@@ -1,223 +1,153 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+  useCallback,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { getTodos, addTodo, deleteTodo } from './api/todos';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { filteredTodosFunction } from './components/Helpers/FilteredTodos';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState(Filter.All);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [title, setTitle] = useState('');
```

Comment:
maybe I would move this state to the header, but I don’t think it’s required

Reply:
If you are using this value in App.tsx, it should be here, if no, then move it to the header.